### PR TITLE
Cleanup config test-empty-config for yamllint

### PR DESCRIPTION
**Pull request by jkupferer**
##### SUMMARY

Add yamllint to test-empty-tonfig and resolve issues.

##### ISSUE TYPE

- Cleanup

##### COMPONENT NAME

Config test-empty-config

### CODE REVIEW

File: ansible/configs/test-empty-config/files/cloud_providers/ec2_cloud_template.j2 [hunk 1]
```diff
@@ -42,246 +42,245 @@ Mappings:
       RHELAMI: ami-cdbdd7a2
```
Comment (fridim) [inline ansible/configs/test-empty-config/files/cloud_providers/ec2_cloud_template.j2:42]: i think we can delete that template and fallback to the default template

**Review by fridim — APPROVED** [review]
